ragna/source_storages/_chroma.py

```diff
@@ -36,19 +36,26 @@ def __init__(self) -> None:
             )
         )
 
-    def _get_collection(self) -> chromadb.Collection:
+    def _get_collection(self, corpus_name: Optional[str]) -> chromadb.Collection:
+        from chromadb.api.types import EmbeddingFunction as ChromaEmbeddingFunction
+
+        if corpus_name is None:
+            corpus_name = self._embedding_id
+
         return self._client.get_or_create_collection(
-            self._embedding_id, embedding_function=self._embedding_function
+            corpus_name,
+            embedding_function=cast(ChromaEmbeddingFunction, self._embedding_function),
         )
 
     def store(
         self,
         documents: list[Document],
         *,
+        corpus_name: Optional[str] = None,
```

Comment:
Ok for now, but we should discuss the name at some point. We can either standardize on one term, e.g. `corpus_name`, for all source storages, or use the specific one, e.g. `collection_name` for Chroma and something else for LanceDB.

Reply:
I don't have any strong opinions here. Happy for you to push whatever change you think is best.